src/ContactPeople/ContactPerson/ContactPersonDetails/ContactPersonDetails.js

```diff
@@ -49,6 +49,12 @@ const ContactPersonDetails = (
           {categories}
         </KeyValue>
       </Col>
+
+      <Col xs={3}>
```

Comment:
Is it enough 3 column for notes?

Reply:
It's a good question. But we have mockup)
![image](https://user-images.githubusercontent.com/43407139/57702294-726d0580-7666-11e9-9d3f-52e15a520f58.png)

Reply:
so it won't use space at the right then. But Note value might be a large one, maybe it makes sense to put `xs={6}` here

Reply:
Ok, will update it